config/routes.rb

```diff
@@ -6,4 +6,5 @@
    root 'welcome#index'
    get 'users/sign_up' => 'users#new'
    post 'users/sign_up' => 'users#create'
+   resources :users
```

Comment:
Ở đây em chỉ cần cho trường hơp show user thôi nên cần khai báo như sau
```
resources :users, only: [:show]
```

Link tham khảo: https://guides.rubyonrails.org/routing.html#restricting-the-routes-created